System.Containers/LocalDocker.cs

```diff
@@ -8,7 +8,7 @@ namespace System.Containers;
 
 public class LocalDocker
 {
-    public static async Task Load(Image x, string name, string baseName)
+    public static async Task Load(Image x, string name, string tag, string baseName)
```

Comment:
```suggestion
    public static async Task Load(Image x, string name, string baseName, string tag = "latest")
```
maybe?

Reply:
I don't hate this but who is going to call it? It was easy enough to require it to be explicitly passed for all the uses I found (so far) + easier to fix.

Reply:
so on my end if I have no tag to supply ($(Version) was empty or something), should I manually pass `latest`?

Reply:
Yeah, that's a safe convention.